elsarticle-template-5-harv.tex

```diff
@@ -94,10 +94,10 @@ \section{Introduction}
     The modules have been built as part of a larger project focusing on automation in kiwifruit orchards.
 
     Automated kiwifruit harvesting and pollination demands computer control, state-of-the-art manipulators, and machine-vision systems.
-    These systems are bulky and have specific geometric requirements dictated by the kiwifruit growing systems.
+    These systems are bulky and have specific geometric requirements dictated by kiwifruit growing systems.
     They share the need for electrical power, air pressure, and a means of locomotion.
     However, they differ in the way they move when operating.
-    The pollinating module moves at a well-known speed with minimum changes in angle.
+    The pollinating module moves at a fixed veloicty with minimum changes in angle.
```

Comment:
A fixed velocity suggests that there are no changes in angle. Since velocity includes direction, my intention was to keep them separate. How about
```suggestion
    The pollinating module moves at a fixed speed with minimum changes in angle.
```

Reply:
I agree.